commcare_connect/opportunity/models.py

```diff
@@ -75,46 +78,95 @@ class Opportunity(BaseModel):
     def __str__(self):
         return self.name
 
+    @property
+    def is_setup_complete(self):
+        return self.paymentunit_set.count() > 0 and self.total_budget
+
+    @property
+    def top_level_paymentunits(self):
+        # payment units that are prereqs of other paymentunits are ignored
```

Comment:
why is this true? both payment units will contribute to the budget right?

Reply:
Refer to https://github.com/dimagi/commcare-connect/pull/298#discussion_r1553284344